olive/passes/onnx/quantization.py

```diff
@@ -217,6 +217,14 @@
             https://onnxruntime.ai/docs/performance/quantization.html for more details on data type selection
         """,
     ),
+    "prepare_qnn_config": PassConfigParam(
+        type_=bool,
```

Comment:
This is only for QNN EP and it should be enabled by default.

Reply:
As discussed offline: exposing the param for all eps since the pass is accelerator agnostic so we don't have accelerator specific config parameters. QNN EP overrides the default values in `_default_config`.